resource-watch/resource-watch
# updates eslint and plugins

## Overview
- updates `eslint` and its plugins.
- removes outdated `eslint-config-vizzuality` and uses `eslint-config-airbnb`.
- removes `eslint-config-prettier`. It wasn't being used.
- removes some old eslint rules.

Note: `eslint-plugin-react` and `eslint` cannot be updated to their latest due to Node's version restrictions.

## Testing instructions
- run `rm -rf node_modules && yarn`
- go to your code editor, linters should be there as usual with displaying new warnings/errors.

## Pivotal task
https://www.pivotaltracker.com/story/show/174977793

---

## Checklist before submitting
